x-pack/plugins/security_solution/public/management/pages/policy/view/components/attack_surface_reduction_form/index.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer, EuiSwitch, EuiText } from '@elastic/eui';
+
+import { OperatingSystem } from '@kbn/securitysolution-utils';
+import { isCredentialHardeningEnabled } from '../../../store/policy_details/selectors';
+import { usePolicyDetailsSelector } from '../../policy_hooks';
+import { ConfigForm } from '../config_form';
+
+const TRANSLATIONS: Readonly<{ [K in 'title' | 'description' | 'label']: string }> = {
+  title: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.details.attackSurfaceReduction.type',
+    {
+      defaultMessage: 'Attack surface reduction',
+    }
+  ),
+  description: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.details.attackSurfaceReduction.explanation',
+    {
+      defaultMessage: 'Options to reduce your attack surface',
```

Comment:
I would vote to remove this message because it is redundant, but will defer to others for specific guidance